src/NUnitTestExtractor/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -183,5 +202,67 @@ public static Level ParseLevel(string levelToParse)
             }
             return level;
         }
+
+        /// <summary>
+        /// Appends all of a TestCase's arguments to a string
+        /// </summary>
+        /// <param name="args">array of arguments to append to string</param>
+        /// <param name="data">the string to add to and also return</param>
+        /// <returns>all of the arguments in a string with proper formatting</returns>
+        public static string AppendArgumentsForTestCasesToString(object[] args, string data)
+        {
+            if(args != null)
+            {
+                int count = args.Length;
+
+                for (int i = 0; i < args.Length; i++)
+                {
+                    var arg = args[i];
+
+                    string argString = null;
+
+                    if (arg != null)
+                        argString = arg as string;
```

Comment:
For consistency, we should put braces around the contents of all if-blocks, even if they're just one line.